package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "com.unity.labs.xrlinerenderer",
+  "displayName": "XR Line Renderer",
+  "version": "0.1.0-preview",
+  "unity": "2019.1",
```

Comment:
@AndrewTHEManeri can/should we support earlier versions?

Reply:
We can - but there is also this github repository for older versions